final_task/rss_parser/rss_parser.py

```diff
@@ -0,0 +1,156 @@
+import json
+import feedparser
+import argparse
+from dataclasses import dataclass
+
+
+@dataclass
+class ArgParser:
+    """class for reading console arguments"""
+    source: str = ""
+    status_version: bool = False
+    status_json: bool = False
+    status_verbose: bool = False
+    limit: int = 1
+
+    arg_parser = argparse.ArgumentParser(description="Pure Python command-line RSS reader.")
+    arg_parser.add_argument("source", type=str, help="RSS URL")
+    arg_parser.add_argument("--version", action="store_true", help="Print version info")
+    arg_parser.add_argument("--json", action="store_true", help=" Print result as JSON in stdout")
+    arg_parser.add_argument("--verbose", action="store_true", help="Outputs verbose status messages")
+    arg_parser.add_argument("--limit", type=int, default=1, help="Limit news topics if this parameter provided")
+    arg_parser_args = arg_parser.parse_args()
+
+    def __post_init__(self):
+        self.source = self.arg_parser_args.source
+        self.status_version = self.arg_parser_args.version
+        self.status_json = self.arg_parser_args.json
+        self.status_verbose = self.arg_parser_args.verbose
+        self.limit = self.arg_parser_args.limit
+
+
+@dataclass
+class Entry:
+    """class for every article from http:link...link.rss"""
+    title: str
+    date: str
+    article_link: str
+    summary: str
+    links: list
+
+    def __post_init__(self):
+        self.parse_html()
+
+    def print_title(self) -> None:
+        print("Title: ", self.title)
+
+    def print_date(self) -> None:
+        print("Date: ", self.date)
+
+    def print_link(self) -> None:
+        print("Link: ", self.article_link)
+
+    def print_summary(self) -> None:
+        print('\n')
+        print(self.summary)
+        print('\n')
+
+    def print_links(self) -> None:
+        print("Links:")
+        for i, link in enumerate(self.links):
```

Comment:
It is better to avoid one-letter variable names, even in such simple constructions.